reactioncommerce/reaction-admin
# Edit Tax sourcing

Resolves #257 , #259 
Impact: **minor**
Type: **feature**

## Issue
1. Tax sourcing was not editable.
2. Addresses were not matched after editing region or postal code in custom tax rate.

## Solution
1. Add field to editor and column to table.
2. Replace emtpy strings with null in postal code and region when creating or updating a custom tax rate.

## Breaking changes
none


## Testing
1. Go to settings/tax-settings
2. Note additional sourcing column
3. Add or edit a custom tax rate, set value for region and postal code
4. Note dropdown for sourcing field
5. Edit a custom tax rate, remove values for region and postal code
6. Note that the edited tax rate is applied when only the country matches
